grails-app/i18n/messages.properties

```diff
@@ -1183,6 +1183,7 @@ inventory.stockTransfers.transferredTo=Transferred To
 inventory.stockTransfers.editCompleted=Cannot edit completed order
 
 stockTransfer.label=Stock Transfer
+stockTransfer.temporaryDisabled.message=Creating Outbound Returns was temporary disabled
```

Comment:
Should be `outboundReturns.temporaryDisabled.message` instead of stock transfer

Reply:
Done